plsync/slices.py

```diff
@@ -142,6 +142,10 @@
                                             ipv6="all",
                                             rsync_modules=['utilization', 'switch']),
 
+    Slice(name="mlab_diff",       index=12, attrs=centos_slice_attrs+web100_enable_attr,
```

Comment:
@functor This slice won't need to have web100 stuff.  So this line can be trimmed a little. If you could amend with another commit, I think we're good.

Reply:
@functor I have another commit to make to operator, so I'm going to add your work to it, and close this PR.